sql_server/pyodbc/base.py

```diff
@@ -157,6 +157,7 @@ class DatabaseWrapper(BaseDatabaseWrapper):
         12: 2014,
         13: 2016,
         14: 2017,
+        15: 2017,
```

Comment:
Should be

```python
        15: 2019,
```

201**9** not 201*7* ;)

But ... this project is no longer being maintained, and you should probably consider switching to https://github.com/FlipperPA/django-mssql-backend which has forked this project, landed the recent suggested improvements here, and is working to get mssql support into Django core. I've ported your PR over to that project (which also needs the change) in FlipperPA/django-mssql-backend#13 . Thanks

Reply:
Ah sorry, I got confused. As noted [here](https://github.com/michiya/django-pyodbc-azure/pull/204#issuecomment-563248221) FlipperPA has actually endorsed the https://github.com/ESSolutions/django-mssql-backend fork which is putting out packages and supports Django 3 now.

I've ported your PR over to that project instead, which is what I should have done in the first place sorry. ESSolutions/django-mssql-backend#32

Reply:
Thanks !